server/cmd/mmctl/commands/post.go

```diff
@@ -103,6 +113,47 @@ func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func postCreateDMCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	ctx := context.TODO()
+	message := args[1]
+	if message == "" {
+		return errors.New("message cannot be empty")
+	}
+
+	user, err := getUserFromArg(c, args[0])
+	if err != nil {
+		return fmt.Errorf("could not find user: %s", err.Error())
+	}
+
+	channel := &model.Channel{
+		Type: model.ChannelTypeDirect,
+		CreatorId: "",
+	}
+	channel.GetOtherUserIdForDM(user.Id)
+	
+	directChannel, _, err := c.CreateChannel(ctx, channel)
```

Comment:
Hello @hanzei, 
Please could you assist me with this issue?
I tried to implement the send DM as a sub-command of the `post create` command such that someone can send a DM by typing the command 
`mmctl post create DM user-name/email  "Hey, there!"`.

I am getting this error when the function tries to create a direct channel

![Screenshot from 2024-09-15 10-41-36](https://github.com/user-attachments/assets/a9130138-6f5e-4260-9cfe-0c95cf2309e7)

It says `Use SaveDirectChannel to create a direct channel` but I cannot seem to locate where the `SaveDirectChannel` method is.

Any input on this?

Reply:
Sure, I'm happy to help!

You need to use `c.CreateDirectChannel` instead of `c.CreateChannel` to create a direct channel.

Reply:
Okay thanks, let  me check